docs/e2e-testing-headscale-derper-tailscale.md

```diff
@@ -68,7 +68,10 @@ User-1, user-2, and jumpbox-2 are on a separate network to the other machines, s
 ssh tailscale-etet-derper -- <<'EOF'
 set -x
 sudo snap install --edge derper
-sudo snap set derper hostname=derper.australiaeast.cloudapp.azure.com
+sudo snap install --edge tailscale
```

Comment:
For testing purposes with https://github.com/canonical/derper-snap/pull/4 , I built the derper snap locally, rsynced it to the tailscale-etet-derper, and installed it:

```
cd derper-snap
snapcraft -v pack
rsync -av derper_1.82.0_amd64.snap tailscale-etet-derper:
sudo snap install ./derper_1.82.0_amd64.snap --dangerous
```